packages/assistify-migrations/server/startup/migrations.js

```diff
@@ -16,7 +16,7 @@ Meteor.startup(() => {
 
 	const usersWithoutName = RocketChat.models.Users.find({name: null}).fetch();
 	usersWithoutName.forEach((user)=>{
-		RocketChat.models.Users.update({_id: user._id}, {$set: {name: _guessNameFromUsername(user.username)}});
+		RocketChat.models.Users.update({_id: user._id}, {$set: {name: _guessNameFromUsername(user.username, user.email)}});
```

Comment:
@jschirrmacher are you sure `user.email` exists? On the DB, it looks as if the property was `user.emails[0].address`, where `user.emails` also needs to be checked for existence

Reply:
You're right - I'll have to change this.